### DISCUSSION TITLE
Invalid token error

### DISCUSSION THREAD

**Discussion by surajrao27** [pallupz/covid-vaccine-booking#235] [Q&A]
What is the scenario when i get the below message and how can i avoid this from happening? (cowin retriever is on )
--------------------------------------------------------------------------------------------------------------------

Token is INVALID.. Next update in 1 seconds. OR press 'Ctrl + C' to refresh now.
Try for a new Token? (y/n Default y): y

--------------------------------------------------------------------------------------------------------------------

**Comment by rishirawal** [comment]
Hi, what is cowin retriever?
